doc/related_projects.rst

```diff
@@ -48,6 +48,9 @@ enhance the functionality of scikit-learn's estimators.
   wrapper around scikit-learn that makes it easy to run machine learning
   experiments with multiple learners and large feature sets.
 
+- `sklearn-deap <https://github.com/rsteca/sklearn-deap>`_ Use evolutionary
```

Comment:
I don't think this belongs in this subsection, which is about providing API wrappers. PyMC appears to be accidentally, incorrectly, in this section.